### DISCUSSION TITLE
Search - Recommendations / Best Practices / Auto-Index

### DISCUSSION THREAD

**Discussion by codebykenny** [nuxt/content#1756] [Q&A] [answered]
I wanted to know how the Nuxt team and others currently maintain their Nuxt Content indexes for search?

@nuxtjs/algolia had an auto-indexer feature that no longer seems to work with nuxt-content 2, it would crawl the content and generate indexes based on the title/description of each page.

Is there an easy drop-in solution that auto-indexes the content for search?

Thanks!

**Comment by Tahul** [comment, marked as answer]
Hello @codebykenny ;

We do not yet have a proper process to do that, I think we use our own crawler but I would have to ask @clemcode or @Atinux for that.

Full text search without additional provider is something we want to explore though.

@clemcode ; could we maybe open source what we use for Nuxt.com search crawling?

Thanks for asking @codebykenny, I hope we'll soon have a proper solution to give you.

**Comment by clemcode** [comment]
Hi! We use [Algolia Docsearch](https://docsearch.algolia.com/) on nuxt.com, so unfortunately no script to share, it is all done directly by the Docsearch crawler...

**Comment by mklueh** [comment]
I'd prefer to have an open API allowing to build different search plugins around it, so nuxt/content would function as a kind of Gridsome, allowing to plug multiple sources in and modify the data in the content layer before it gets exposed to the query API.
